### DISCUSSION TITLE
Change the media save location

### DISCUSSION THREAD

**Discussion by nohsor** [RandomNinjaAtk/docker-radarr-extended#33] [Q&A] [answered]
Hi, thank you very much for your efforts on this radarr modification.
I have question regarding the media (Trailer, Behind the scenes, Featurettes).
Is there a method to make them saved in one folder (say "extras" folder)?
I used Kodi as media player, and it can only use extra multiple media if they are placed in "extra" folder.

I tried to change the code in the script "MovieExtras.bash", both from inside and outside the container 
from:
        if [ "$tmdbExtraType" == "Featurette" ]; then
            extraFolderName="featurettes"
        elif [ "$tmdbExtraType" == "Trailer" ]; then
            extraFolderName="trailers"
        elif [ "$tmdbExtraType" == "Behind the Scenes" ]; then
            extraFolderName="behind the scenes"

to:
        if [ "$tmdbExtraType" == "Featurette" ]; then
            extraFolderName="extras"
        elif [ "$tmdbExtraType" == "Trailer" ]; then
            extraFolderName="extras"
        elif [ "$tmdbExtraType" == "Behind the Scenes" ]; then
            extraFolderName="extras"
however, after starting the container, the code revert to the original values.

I ran the app in a contianer with openmediavault using stack compose of portainer.
Thank you very much.

**Comment by RandomNinjaAtk** [comment, marked as answer]
There isn't a way to modify the script, you can make a copy and move it to a different location and modify to your hearts content.
